google/cloud/bigquery/table.py

```diff
@@ -1023,6 +1024,25 @@ def table_constraints(self) -> Optional["TableConstraints"]:
             table_constraints = TableConstraints.from_api_repr(table_constraints)
         return table_constraints
 
+    @property
+    def resource_tags(self):
+        """Dict[str, str]: Resource tags for the table.
+
+        See: https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#Table.FIELDS.resource_tags
+
+        Raises:
+            ValueError: For invalid value types.
+        """
+        return self._properties.setdefault(
```

Comment:
```suggestion
        return self._properties.get(
```

Use `get()` just for consistency.

Reply:
Hi @Linchin, thank you for your review.
I have a question regarding `resource_tags` that I'd like to discuss further.

Given that `resource_tags` is a dictionary, similar to `labels`, should we consider a similar implementation approach?

https://github.com/googleapis/python-bigquery/blob/9c504186f03ffb8b86836c683912b310db2435e9/google/cloud/bigquery/table.py#L1220-L1228

Using `get()` with `{}` as the default works well for:

```python
table.resource_tags = {"key": "value"}
```

But I'm not sure if it would properly handle:

```python
table.resource_tags["key"] = "value"
```

What are your thoughts?

Reply:
Thanks for pointing this out, indeed `get()` with default value will not handle the second case you provided. Let me undo this change.